croquemort/crawler.py

```diff
@@ -51,23 +43,24 @@ def check_url(self, url_group_frequency):
             self.storage.store_group(url, group)
             if frequency:
                 self.storage.store_frequency(url, group, frequency)
+        head_timeout = self.config.get('CRAWLER_HEAD_TIMEOUT', HEAD_TIMEOUT)
+        get_timeout = self.config.get('CRAWLER_GET_TIMEOUT', GET_TIMEOUT)
+        no_head_domains = self.config.get('KNOWN_HEAD_OFFENDER_DOMAINS', [])
         try:
```

Comment:
Not related to that PR but that `try… except` block is quite heavy.

Reply:
Yes :-(